app/views/items/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>All Items</h1>
+
+<% @items.each do |item| %>
+<p> Item Name: <%= item.name %> </p>
+<p> Item price: <%= item.price %> </p>
+<p> Item Name: <%= item.supermarket.name %> </p>
+<p> Purchased by </p>
+<% item.customers.each do |customer| %>
+  <p> <%= customer.name %> </p>
```

Comment:
Be careful with the details of the user stories! US3 asks for the *count* of customers rather than listing all their names. We would've wanted to see you encapsulate a `customer_count` method in the items model to achieve this